mantidproject/mantid
# Remove old save options and reorder them

Removes SaveCSV and SaveNexusProcessed and reorders the save options, as requested by Steve.

Fixes #18334

## To test

1. Open the ISIS SANS GUI
2. Select the LOQ instrument
3. Load the user file: USER_LOQ_161G_Hedstrom_8mm_Magnet-MERGED.txt
4. Set   LOQ97072.nxs as the sample scatter input
5. Press Load Data
6. Press 1D Reduce
7. In the save group box, check `RKH`, `NXcanSAS` adn `CanSAS`
8. Press `Save Result`
  * Confirm that the results are saved without errors (make sure that they were saved on your file system)
9. Click `Save Other` and save dialog will appear
10. Select `RKH`, `NXcanSAS` adn `CanSAS` as the algorithm option, select `97074main_1D_2.2_10.0` as the workspace to save by clicking on it
11. Press `Save`
  * Confirm that the results are saved without errors (make sure that they were saved on your file system)


## Release notes


See [here](https://github.com/mantidproject/mantid/pull/18337/commits/f125cda6e3bb978bdf52ccf180750567b8b97486)
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
